core/src/main/java/org/p2p/core/token/Token.kt

```diff
@@ -78,20 +79,15 @@ sealed class Token constructor(
                 isZero &&
                 visibility == TokenVisibility.DEFAULT
 
-        fun getFormattedUsdTotal(): String? = totalInUsd?.asUsd()
+        fun getFormattedUsdTotal(includeSymbol: Boolean = true): String? {
+            return if(includeSymbol) totalInUsd?.asUsd() else totalInUsd?.formatFiat()
+        }
 
         fun getFormattedTotal(includeSymbol: Boolean = false): String =
             if (includeSymbol) {
-                "${total.formatToken()} $tokenSymbol"
```

Comment:
strange diff, result looks like this
```
        fun getFormattedTotal(includeSymbol: Boolean = false): String =
            if (includeSymbol) {
                "${total.formatToken(decimals)} $tokenSymbol"
            } else {
                total.formatToken(decimals)
            }
```

please verify @kabramovp2p 
we were not using decimals, is this a bug?